src/presentation/ui/component/form/passwordInput.templ

```diff
@@ -0,0 +1,35 @@
+package componentForm
+
+templ PasswordInput(
+	id, label, bindValuePath string,
+	denseMode bool,
+) {
+	<!-- PasswordInput -->
+	<div class="relative w-full" x-data="{ isPasswordReadable: false }">
+		<input
+			x-bind:type="isPasswordReadable ? 'text' : 'password'"
+			id={ id }
+			name={ id }
+			x-model:value={ bindValuePath }
```

Comment:
Is this working? You created a new x-data, so probably the state here will break. By the way, why do you need Alpine to hold the is readable state? What that even means?

Reply:
This was already discussed on Slack, and we agreed on this code.